force-app/main/default/classes/RecentServiceSessionController.cls

```diff
@@ -0,0 +1,23 @@
+/*
+ *
+ *  * Copyright (c) 2020, salesforce.com, inc.
+ *  * All rights reserved.
+ *  * SPDX-License-Identifier: BSD-3-Clause
+ *  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ *
+ */
+public with sharing class RecentServiceSessionController {
+    @TestVisible
+    private static ServiceSessionService serviceSessionService = new ServiceSessionService();
+
+    @AuraEnabled(cacheable=true)
+    public static Map<String, List<ServiceSession__c>> getServiceSessionsByStartDate(
+        String dateLiteral
+    ) {
+        try {
+            return serviceSessionService.getServiceSessionsByStartDate(dateLiteral);
```

Comment:
I was thinking about this over the weekend and to be more secure from injection we should create a constant of allowed date literals and verify that the set contains the date literal before actually doing the query. Let me know what you think.

Reply:
lol @jjbennett !! I was thinking about it this morning since this sprint there is a story to add last_week as another option. I will make that change this week. Also I was thinking the way I overloaded the addAttendenceData in testDataFactory is not correct either. I need to see if we can pass the date instead of just the frequency. Will think more about that refactor too